ambientweather2sqlite/daemon.py

```
@@ -8,7 +8,7 @@
 from ambientweather2sqlite.awparser import extract_values
 from ambientweather2sqlite.server import Server
 
-from .database import insert_observation
+from .database import DatabaseManager
 from .metadata import create_metadata
```

Flake8 (7.2.0)

[error] '.metadata.create_metadata' imported but unused

(F401)